exonum-java-binding/core/rust/integration_tests/tests/service_runtime.rs

```diff
@@ -109,10 +75,16 @@ fn load_nonexistent_artifact() {
 #[test]
 #[should_panic(expected = "Unable to load artifact")]
 fn load_artifact_twice() {
-    let runtime = get_runtime();
-    let artifact_path = create_service_artifact_valid(runtime.get_executor());
-
-    runtime.load_artifact(&artifact_path);
+    let res = catch_unwind(|| {
+        let runtime = get_runtime();
+        let artifact_path = create_service_artifact_valid(runtime.get_executor());
+        runtime.load_artifact(&artifact_path);
+        (runtime, artifact_path)
+    });
+    assert!(res.is_ok());
+
+    let (runtime, artifact_path) = res.unwrap();
```

Comment:
Wouldn't it be more natural and easier to read if the approach earlier recommended was used? Why these unnatural unwraps and redundant checks of the code that is _not_ being tested?

```
fn assert_panics(error_message_pattern: String, f: Callable) {
  let res = catch_unwind(f.call());
  assert(res.is_err());
  assert_matches|equals|starts_with(res.get_message(), error_message_pattern);
}
```

Reply:
Now I get what you mean. Definitely, it looks better.